datalad_xnat/update.py

```diff
@@ -79,13 +83,14 @@ class Update(Interface):
             args=("-f", "--force",),
             doc="""force (re-)building the addurl tables""",
             action='store_true'),
-        **_XNAT.cmd_params
+        **_XNAT.cmd_params,
+        jobs=jobs_opt
```

Comment:
```suggestion
        jobs=jobs_opt,
        **_XNAT.cmd_params
```

With some Python versions it is better (needed) to have the kwargs last.